system-int-test/test/cmr/system_int_test/search/autocomplete/suggestion_reindex_test.clj

```diff
@@ -76,8 +90,29 @@
                             (fu/processing-level-id "PL1")
                             {:DataCenters [(data-umm-spec/data-center {:Roles ["ARCHIVER"] :ShortName "DOI/USGS/CMG/WHSC"})]
                              :ScienceKeywords [(:ScienceKeywords (fu/science-keywords sk1 sk2))]})
+        coll3 (d/ingest-concept-with-metadata-file "CMR-6287/C1000000029-EDF_OPS.xml"
+                                                         {:provider-id "PROV1"
+                                                          :concept-type :collection
+                                                          :format-key :echo10})
         _ (index/wait-until-indexed)]
 
     (index/reindex-suggestions)
     (index/wait-until-indexed)
-    (search/get-autocomplete-suggestions "q=level2")))
+    (testing "Ensure that response is in proper format and results are correct"
+      compare-autocomplete-results
```

Comment:
I don't think this has the expected and results being passed into it. Can you confirm? I had to add '()' aroudn it and the following arguments to get it run

Reply:
Oh yeah looks like parinfer bit me. Fixed